### PR TITLE
Board page

### PR DESCRIPTION
**Pull request by josh-jacobsen**
If merged this pull request will add a board page and display all the statuses in the store. 

Close button added but not linked up yet.

### CODE REVIEW

File: client/actions.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const SHOW_LANDING_PAGE = 'SHOW_LANDING_PAGE'
+
+export const showLandingPage = () => {
+  return {
+    type: SHOW_LANDING_PAGE
+  }
+}
```
Comment (mixmix) [inline client/actions.js]: I don't think these have been used, please remove

File: client/reducer.js [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
+    id: 2,
+    name: 'Jane'
+  } ],
+  statuses: [ {
+    teamMember: {
+      id: 2,
+      name: 'Jane',
+    },
+    message: 'yeah ok'
+  }]
```
Comment (mixmix) [inline client/reducer.js:13]: we should remember to remove this in the future

Comment (josh-jacobsen) [inline client/reducer.js:13]: agreed

File: client/actions.js (hunk 1 above)
Comment (josh-jacobsen) [inline client/actions.js]: done

File: test/reducerTest.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import test from 'ava'
+
+import { SHOW_LANDING_PAGE } from '../client/actions'
```
Comment (mixmix) [inline test/reducerTest.js]: this needs to be removed now too

Comment (josh-jacobsen) [inline test/reducerTest.js]: done - deleted file as it was not working and not being used anywhere

**Pull request merged**